### ISSUE TITLE
Is the project still maintained?

### ISSUE THREAD

**Issue by stsrki** [ray440/IcuBlazor#13]
Hi @ray440 

I found this testing project quite interesting, mainly because it supports the testing of components that involves JSInterop. But before I commit myself, and my project, to use IcuBlazor I have a concern over its future. From what I can see no work was done since November. So my question is, do you still plan to work on it and improve it? Or is it abandoned? Which would be a shame really.

**Comment by ray440** [comment]
Yes, IcuBlazor is in active development!  The github repo... no so much. There doesn't seem to be anyone interested in maintaining the code other than us.  If we get enough demand for source, we can update the repo.

The latest versions (1.4.3 Feb'22) are in the nuget packages.  Please use those.; there's some really cool stuff there.

**Comment by stsrki** [comment]
Well, I believe if you give it enough traction then more people will use and in time they will contribute. Currently, even if they wish to try they come to the repository, and see that it is built on NET5(even though the NuGet is on NET6), with small numbers of stars, and almost no word from the maintainers. It is not a good message to them.

I know firsthand that working with a community is needed if any project wishes to succeed as an OSS.

**Comment by ray440** [comment]
You are correct. We haven't done much marketing & outreach yet.  I just realized you're working on Blazorise. Nice work!  Maybe I can get some advice from you offline? (ray440@gmail.com)

**Comment by stsrki** [comment]
> Maybe I can get some advice from you offline?

Yeah, sure. I'm always open for talk :)
